packages/web-forms/src/components/controls/Input/Geopoint/GeolocationRequestDialog.vue

```diff
@@ -123,7 +123,7 @@ onBeforeUnmount(cleanup);
 		<template #header>
 			<div class="geo-dialog-header">
 				<div class="geo-dialog-header-title">
-					<ProgressSpinner class="spinner" stroke-width="4" />
+					<ProgressSpinner class="geo-spinner" stroke-width="4" />
```

Comment:
Conflicts with Central's class names.

This code is loaded in a dialog. And the dialogs are at the end of the DOM, outside the Web Forms super class. 
This is the fastest fix. We need to scope that later.